doc/architecture/decisions/0015-container-api.md

```diff
@@ -7,39 +7,39 @@ Date: 2018-09-17
 Accepted
 
 ## Glossary
-* **Container**: *We have been struggling to find a term for this concept that feels fitting.
-  In the following, we are using the word *Container*, knowing that this is a temporary placeholder term
+* **Conductor**: *We have been struggling to find a term for this concept that feels fitting.
+  In the following, we are using the word *Conductor*, knowing that this is a temporary placeholder term
   and might be replaced in the future by something else (like *Composer*, *Orchestrator*, *Organism*,
   *Holochain System*, ...).*
 
-  A *Container* contains Holochain instances - in the general case: multiple Holochain instances, of which
+  A *Conductor* contains Holochain instances - in the general case: multiple Holochain instances, of which
   each run exactly one given DNA.
   It makes use of the [core_api](core_api/src/lib.rs) and
-  thus manages (creates, deletes, starts, stops) instances. The Container is naturally
+  thus manages (creates, deletes, starts, stops) instances. The Conductor is naturally
   mediating between any kind of user and all installed Holochain apps (here: app = Holochain instance =
-  embodied DNA = *Phenotype*).  The Container is necessarily the only architectural piece that has unique
+  embodied DNA = *Phenotype*).  The Conductor is necessarily the only architectural piece that has unique
   and direct access to the [core_api](core_api/src/lib.rs) and thus the capabilities and zome functions
   of Holochain apps.
 
   **Definition**: Any software module (be it a library, network service or executable run by end-users)
   that uses Holochain's [core_api](core_api/src/lib.rs) to instantiate
-  a Holochain instance is called *Container*. It is the necessary layer that *contains* an instance,
+  a Holochain instance is called *Conductor*. It is the necessary layer that *contains* an instance,
   as [core_api](core_api/src/lib.rs) being a library it (and instances created by it)
   can not exist as a process themselves.
 
 ## Context
 
-* We have several different roles for using a Holochain app (through a *container*):
+* We have several different roles for using a Holochain app (through a *conductor*):
   * local, QML based UI components
   * as a special case of the above: administration UI
   * externally located (living in a browser) web UIs connected through some form of IPC
   * *bridged* apps, i.e. Holochain app as the user of another Holochain app
   * Services hosting multiple Holochain apps and exposing their zome functions to UIs selectively
-* With HoloSqape and the Holochain-nodejs, we already have two container implementations
+* With HoloSqape and the Holochain-nodejs, we already have two conductor implementations
```

Comment:
This line should change to mention the renamed Rust container, and the renamed nodejs container